crowdin/messages.json

```diff
@@ -1 +1 @@
-1202105916":"Interim | Affiliate","-250622057":"Our improved bot builder at our new home","-742796250":"You read right. We are rebranding Binary.com to Deriv.com.","-7818514":"Here’s why: In 2019, we celebrated our 20th year anniversary. Inspired by your love for online trading, we decided to create something new as part of our renewed commitment to making online trading effortless and accessible for you.","-720766028":"You inspired us","-580326996":"to create something new","-110329645":"In 2019, we celebrated our 20th year anniversary. Inspired by your love for online trading, we decided to create something new as part of our renewed commitment to making online trading effortless and accessible for you.","-754514685":"Love trading on Binary.com’s signature platform?","-293239181":"Interim | Deriv","-1924436660":"Trade on MT5 at our new home","-844707100":"Try MT5 on Deriv.com","-1798345294":"After 20 years in the market, we are stepping into the third decade of operations with a renewed commitment to you, our clients. Our new name and platforms are in line with our vision to provide a broader range of products and an enhanced user experience. With the rebranding, our intention is to continue serving you with simple, flexible, and reliable online trading solutions.","-1676645687":"The rebranding involves a new name, logo, corporate colours, and website which you may have already seen in our online presence. You will also enjoy a wider range of instruments to trade with, besides the availability of 4 advanced platforms on a single trading space. Over time, we will roll out more new solutions, all geared towards improving your trading experience.","-804933634":"No, Deriv.com is built, owned, and managed by the same team that brought you Binary.com. We like to think of Deriv.com as the next step in the evolution of our company. We have taken lessons from 20 years of experience to produce a diverse range of products that give clients the flexibility and ease to trade according to their preference with confidence.","-35835337":"Deriv.com gives you multiple trading options in an environment that is simple, flexible, and reliable. Three core products — DTrader, DBot, and DMT5 — are offered alongside SmartTrader, the popular trading platform from Binary.com.","-960630757":"As we roll out new features in phases, you can look forward to such things as:","-1754226645":"Multipliers that potentially offer traders higher profits while limiting risks","-24345835":"Yes. Deriv.com and Binary.com are owned by the same group, which is licensed and regulated in many jurisdictions around the world.","-367386425":"The Binary.com website will still be around for a while. In due time, we will phase out Binary.com. However, your accounts, funds, and everything you have on Binary.com will be available and accessible by you on Deriv.com.","-952281983":"We’ve planned this rebranding process carefully and over many months with you, our clients, in mind. Based on the experi
```

Comment:
can revert this change to not make a conflict with crowdin branch later on when rebasing